src/ImageSharp/Formats/Gif/LzwDecoder.cs

```diff
@@ -245,7 +250,7 @@ public void SkipIndices(int minCodeSize, int length)
         // It is possible to specify a larger LZW minimum code size than the palette length in bits
         // which may leave a gap in the codes where no colors are assigned.
         // http://www.matthewflickinger.com/lab/whatsinagif/lzw_image_data.asp#lzw_compression
-        if (minCodeSize < 2 || clearCode > MaxStackSize)
+        if (minCodeSize < 2 || minCodeSize > MaximumLzwBits || clearCode > MaxStackSize)
```

Comment:
```suggestion
        if ((uint)minCodeSize - 2 > MaximumLzwBits - 2 || clearCode > MaxStackSize)
```
or move this check into a helper, than it's more self-documenting what's going on.